### PR TITLE
[hailtop.utils]test for grouped

### PR DESCRIPTION
**Pull request by CDiaz96**

### CODE REVIEW

File: hail/python/test/hailtop/utils/test_utils.py [hunk 1]
```diff
@@ -126,3 +127,58 @@ def test_parse_docker_image_reference():
     assert x.digest is None
     assert x.name() == 'us-docker.pkg.dev/my-project/my-repo/test-image'
     assert str(x) == 'us-docker.pkg.dev/my-project/my-repo/test-image'
+
+
+
```
Comment (danking) [inline hail/python/test/hailtop/utils/test_utils.py:132]: only two spaces between function definitions please!

File: hail/python/hailtop/utils/utils.py [hunk 2]
```diff
@@ -108,6 +108,8 @@ def digits_needed(i: int):
 
 
 def grouped(n, ls):
+    if n < 1:
+        raise ValueError('invalid value for n')
```
Comment (danking) [inline hail/python/hailtop/utils/utils.py:112]: Let's also include the bad `n` value in the error message. You can use a [Python format string](https://docs.python.org/3/tutorial/inputoutput.html#fancier-output-formatting) to include the number `n` in the string itself.

**Review by danking — CHANGES_REQUESTED** [review]

File: hail/python/test/hailtop/utils/test_utils.py (hunk 1 above)
Comment (CDiaz96) [reply hail/python/test/hailtop/utils/test_utils.py:132]: Fixed

**Review by danking — APPROVED** [review]
Looks great!

**Pull request merged**